# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xmms/gcc-4.0.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual gcc-4.0 fixes.

  - Rene Rebe <rene@exactcode.de>

--- xmms-1.2.10/General/ir/ir.c	2001-03-05 13:17:44.000000000 +0000
+++ xmms-1.2.10-fixed/General/ir/ir.c	2005-09-03 15:44:41.000000000 +0000
@@ -16,10 +16,10 @@
 #include "ir.h"
 
 /* Important stuff to know */
-static gboolean keepGoing = FALSE;
+gboolean keepGoing = FALSE;
 
 /* The thread handle */
-static pthread_t irapp_thread;
+pthread_t irapp_thread;
 
 /* Declarations for calls that we need to mention in the plugin struct */
 static void init(void);
